models/balances/bitcoin/latest_balances_bitcoin_sataoshi_day.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(
+        schema = 'balances_bitcoin',
+        alias = 'satoshi_latest_day',
+        materialized='table',
```

Comment:
with the upstream spell `balances_bitcoin_satoshi_day` being a view, materializing this as a table is slow and crashing the cluster. i wonder if we need to revisit that other spell to materialize as well, if the plan is to build on top of it.

Reply:
the issue with `balances_bitcoin_satoshi_day` being made incremental came down to the CTE which builds a sequence over time, then obtaining values to match those dates. it wasn't straightforward to build.

Reply:
we have an ongoing effort to build transfers / balances on evm blockchains, where the approach may make sense to mimic for bitcoin.

you can find the new approaches here:
https://github.com/duneanalytics/spellbook/tree/main/models/_sector/tokens/ethereum

Reply:
Ok I will keep an eye out on this new approach. seems like we will need to refactor this later on